account_invoice_change_currency/data/ir_server_actions.xml

```diff
@@ -0,0 +1,18 @@
+<odoo noupdate="1">
```

Comment:
No need to remove XML header.

Same for other XML files

Reply:
Done!